### PR TITLE
os/filestore: get com_lock in op_apply_start

### PR DESCRIPTION
**Pull request by XinzeChi**
Signed-off-by: Xinze Chi <xinze@xsky.com>

### CODE REVIEW

**Comment by XinzeChi** [comment]
@liewegas ,do you think we need get lock here?

**Comment by liewegas** [comment]
If it is really possible to race then we should just drop the assert.  What is the scenario you're worried about?

**Comment by XinzeChi** [comment]
I also think drop the assert is ok.

File: src/os/filestore/JournalingObjectStore.cc [hunk 1]
```diff
@@ -130,7 +130,6 @@ uint64_t JournalingObjectStore::ApplyManager::op_apply_start(uint64_t op)
   dout(10) << "op_apply_start " << op << " open_ops " << open_ops << " -> "
 	   << (open_ops+1) << dendl;
   assert(!blocked);
-  assert(op > committed_seq);
```
Comment (amitkumar50) [inline src/os/filestore/JournalingObjectStore.cc]: Agree with @liewegas it would be good to add `race condition` for future reference.

**Pull request closed without merging**